packages/backend/src/index.ts

```diff
@@ -39,6 +40,9 @@ export function Clerk(options: ClerkOptions) {
   return {
     ...apiClient,
     ...requestState,
+    authenticateRequest: (request: Request, requestOptions: AuthenticateRequestOptions) => {
```

Comment:
Let's propagate this change to `request.ts` to avoid this override. It makes more sense to have this signature.